happi/backends/json_db.py

```diff
@@ -168,12 +173,12 @@ def _iterative_compare(self, comparison):
             except Exception as ex:
                 logger.debug('Comparison method failed: %s', ex, exc_info=ex)
 
-    def get_by_id(self, id_):
+    def get_by_id(self, _id: str) -> ItemMeta:
```

Comment:
`id_` was because the name clashed with Python's builtin `id()`, not so much naming it consistently with the database key

Reply:
I was wondering about that.  Good to know